cfgov/v1/models/molecules.py

```diff
@@ -39,7 +39,7 @@ class Meta:
 class TextIntroduction(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     intro = blocks.CharBlock(max_length=100, required=True)
-    body = blocks.RichTextBlock(blank=True)
+    body = blocks.RichTextBlock(required=False)
```

Comment:
According to [GHE]/flapjack/Modules-V1/wiki/Text-Introduction body should be required. Check with @ajbush or @duelj about whether this is correct on the wiki though.